include/flecs/addons/cpp/mixins/entity/builder.hpp

```diff
@@ -394,6 +394,35 @@ struct entity_builder : entity_view {
         return this->override<First>(_::cpp_type<Second>::id(this->m_world));
     }
 
+    /** Mark id for auto-overriding.
+     * When an entity inherits from a base entity (using the IsA relationship)
+     * any ids marked for auto-overriding on the base will be overridden
+     * automatically by the entity.
+     *
+     * @param id The id to mark for overriding.
+     */
+    Self& override_second(flecs::id_t id) {
+        return this->add_second(ECS_OVERRIDE | id);
```

Comment:
I don't think this works, the `add_second` method requires a template argument:
```cpp
    template<typename Second>
    Self& add_second(flecs::entity_t first) {
        return this->add(first, _::cpp_type<Second>::id(this->m_world));
    }
```

Reply:
I'm not sure what I should be passing over to add_second 🤔

Reply:
Probably something like that:
```cpp
    template <typename Second>
    Self& override_second(flecs::id_t id) {
        return this->add_second<Second>(ECS_OVERRIDE | id);
    }
```